src/lib/components/WebvizPluginPlaceholder.react.js

```diff
@@ -33,13 +33,9 @@ export default class WebvizPluginPlaceholder extends Component {
     }
 
     componentDidUpdate(_, prevState) {
-        if (this.props.zip_base64 !== "") {
-            const now = new Date();
-            const filename = `webviz-data-${now.getFullYear()}-${now.getMonth() +
-                1}-${now.getDate()}.zip`;
-
-            download_file(filename, this.props.zip_base64);
-            this.props.setProps({ zip_base64: "" });
+        if (JSON.stringify(this.props.download) !== "{}" && this.props.download !== "") {
+            download_file(this.props.download.filename, this.props.download.content);
```

Comment:
Should we now also generalize MIME type of downloaded file?

I.e. add a third argument in this call, which then is used in 
https://github.com/equinor/webviz-core-components/blob/6a9caadd4ef5e8a601b140d07a6938fd632a431f/src/lib/private-components/webviz-plugin-placeholder-resources/download_file.js#L7
to get rid of currently hardcoded `text/plain` (this could then be set to e.g. `text/csv` or one of the other [common MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) by the plugin author).

_Syntax detail: Could also consider passing in the whole `this.props.download` prop/object now in the call, and use the technique [here](https://medium.com/@afontcu/cool-javascript-9-named-arguments-functions-that-get-and-return-objects-337b6f8cfa07). That would then perhaps make it easier to still support the call in
https://github.com/equinor/webviz-core-components/blob/6a9caadd4ef5e8a601b140d07a6938fd632a431f/src/lib/components/WebvizPluginPlaceholder.react.js#L90-L94
which uses a different set of arguments (MIME type is stored within the browser generated BLOB, and available as `blob.type`)._